### DISCUSSION TITLE
Expandable sections in the page outline.

### DISCUSSION THREAD

**Discussion by joshmarinacci** [GitbookIO/community#474] [Feature Requests]
I'd like to use expandable sections for our FAQ, but they don't appear in the page outline the way top level H1s and H2s do. Is there a setting that will let them be in the page outline?

**Comment by aikoii** [comment]
👋 @joshmarinacci, currently, there is no way to have expandables appear in the page outline - but thank you for sharing your use case 👍

**Comment by hansiruting** [comment]
I am looking for this feature too. Waiting for it.
